repository/comment.go

```diff
@@ -7,4 +7,5 @@ import "github.com/openhacku-saboten/OmnisCode-backend/domain/entity"
 type Comment interface {
 	FindByPostID(postid int) (comments []*entity.Comment, err error)
 	Insert(comment *entity.Comment) error
+	FindByID(postid, commentid int) (comment *entity.Comment, err error)
```

Comment:
他のところもIDは大文字で統一しているので、統一しても良いかなと思いました。

```suggestion
	FindByID(postID, commentID int) (comment *entity.Comment, err error)
```

Reply:
命名が適当でした......すみません
idが小文字になっているところが他にもあるので合わせて修正します．